airflow/www/views.py

```diff
@@ -4203,7 +4203,8 @@ class TaskInstanceModelView(AirflowPrivilegeVerifierModelView):
     class_permission_name = permissions.RESOURCE_TASK_INSTANCE
     method_permission_name = {
         'list': 'read',
-        'action_clear': 'edit',
+        'action_clear': 'delete',
```

Comment:
This seems like this should be edit, like the `set_*`'s. I know you mirrored DagRunModelView, but I'm wondering if this is right and DagRunModelView is wrong 🤷‍♂️.

Reply:
I think you have a valid point here.

Reply:
Now that I'm thinking about it, you're definitely right.  This makes sense to me and I'm going to make the change.

Reply:
Can you update the PR description too?